### DISCUSSION TITLE
Request to add a variable in the settings which is responsible for the number of characters in desktop icon name strings

### DISCUSSION THREAD

**Discussion by yaaaaa** [linuxdeepin/developer-center#3367] [Features Request & Ideas | 特性请求 & 头脑风暴]
Request to add a variable in the settings which is responsible for the number of characters in desktop icon name strings

**Comment by raspin0** [comment]
Sounds completely useless. Why would you change the maximum value of available characters. Are 256 characters not enough? Why would you reduce it?

**Reply by yaaaaa** [reply]
Sorry for my Bad English. I meant the space for the number of characters in each line in the name, one line does not get as many characters as I would like. With 1920x1080 and a font size of 13, it holds 10 characters, the rest is carried on the next line, and if for languages such as Chinese it looks okay, but for languages with longer words, it does not look very nice.

**Comment by BLumia** [comment]
What's the actual use case? Adding setting options to everything is not a good idea so you need a reason for it. I can't think of a good reason for this option...
